### DISCUSSION TITLE
Webmin on alpine-vaultwarden

### DISCUSSION THREAD

**Discussion by emddummi** [tteck/Proxmox#1973] [Help Needed] [answered]
Hi,

is it possible to Make the Script for webmin enable to install into alpine-vaultwarden lxc?

I get error

`bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/misc/webmin.sh)"
wget: unrecognized option: L`

**Comment by tteck** [comment, marked as answer]
The script will not install on Alpine. You can install Webmin manually

```
apk update
apk add perl perl-net-ssleay
cd /opt
wget -O - https://github.com/webmin/webmin/archive/refs/tags/2.104.tar.gz | tar -xzf -
mv webmin-2.104 webmin
cd webmin
./setup.sh /usr/local/webmin
```
